apps/src/applab/designMode.js

```diff
@@ -473,7 +474,8 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging, prefix) {
   if (!Applab.levelHtml) {
     return;
   }
-  var levelDom = $.parseHTML(Applab.levelHtml);
+  var sanitized = sanitizeHtml(Applab.levelHtml);
```

Comment:
I agree that there is something aesthetically pleasing about rejecting rather than sanitizing level html written by a hacker, but is there any other advantage of failing in that case? Both provide the same level of security against hackers. Neither has an advantage for legitimate users when the app is working properly. So the only difference is that if there is a programming error, we would block all `levelHtml` for legitimate users rather than allowing the legitimate part of it through.